pkg/generators/markers.go

```diff
@@ -0,0 +1,217 @@
+package generators
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"k8s.io/gengo/types"
+)
+
+// Subset of spec.SchemaProps that may be placed into a marker comment
+type ValueValidations struct {
```

Comment:
Shouldn't we use this: https://github.com/kubernetes/kube-openapi/blob/master/pkg/validation/spec/schema.go#L161-L197?

Reply:
Yeah probably can. I think this was used because original implementation meant to serialize the fields and we wanted to exclude missing fields. But I don't think that's a problem anymore. 

@Schnides123 remove ValueValidations and use schemaprops again?